src/CustomElementInternals.js

```diff
@@ -4,7 +4,7 @@ import CEState from './CustomElementState.js';
 export default class CustomElementInternals {
   constructor() {
     /** @type {!Map<string, !CustomElementDefinition>} */
-    this._localNameToDefinition = new Map();
+    this._nameToDefinition = new Map();
```

Comment:
For autonomous custom elements, the custom element name is the same as `localName`. But for customized builtin elements, the `localName` is different than the name of the custom element. I have changed this Map so that it looks thing up by `name` instead of `localName`, so that it works with both autonomous and customized builtin elements.